src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -21,45 +34,339 @@ const ActorInit Bg_Spot15_Rrbox_InitVars = {
     (ActorFunc)BgSpot15Rrbox_Update,
     (ActorFunc)BgSpot15Rrbox_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B3960.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B39E8.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+Vec3f D_808B45C4[] = {
+    { 770.0f, 1490.0f, -299.0f },
+    { 770.0f, 1550.0f, -299.0f },
+};
+
+Vec3f D_808B45DC[] = { { 29.99f, 0.01f, -29.99f },
+                       { -29.99f, 0.01f, -29.99f },
+                       { -29.99f, 0.01f, 29.99f },
+                       { 29.99f, 0.01f, 29.99f },
+                       { 0.0f, 0.01f, 0.0f } };
+
+s32 D_808B4618[] = { 0, 0 };
+
+void func_808B3960(BgSpot15Rrbox* this, GlobalContext* globalCtx, s32* arg2, s32 flags) {
+    s32 pad;
+    s32 sp30 = 0;
+    u32 pad2;
+
+    DynaPolyInfo_SetActorMove(&this->dyna, flags);
```

Comment:
cf `DynaPolyMoveFlag`'s prototype, flags is of type `DynaPolyMoveFlag` (effectively a s32 but it's a enum)

and the 0 passed as flags to `func_808B3960` is `DPM_UNK` per that enum